src/transformers/training_args.py

```diff
@@ -963,6 +964,19 @@ class TrainingArguments:
             )
         },
     )
+    timeout: Optional[int] = field(
```

Comment:
```suggestion
    ddp_timeout: Optional[int] = field(
```
Let's make it clear this is a DDP argument.